nrich-validation/src/main/java/net/croz/nrich/validation/constraint/validator/MinDateValidator.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright 2020-2023 CROZ d.o.o, the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package net.croz.nrich.validation.constraint.validator;
+
+import net.croz.nrich.validation.api.constraint.MinDate;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.time.LocalDate;
+
+public class MinDateValidator implements ConstraintValidator<MinDate, LocalDate> {
```

Comment:
LocaleDate is not enough we should support other date types too (i.e. Date, Instant, ZonedDateTime etc), also users should be able to specify format.